tests/API/XCCDF/unittests/test_xccdf_oval_absolute_path.sh

```diff
@@ -41,7 +50,15 @@ function testInDirectory(){
 }
 
 name=$(basename $0 .sh)
-ORIGINAL_XCCDF="$(readlink -e "$srcdir/${name}.xccdf.xml")"
+
+case $(uname) in
```

Comment:
I think that we can stick with `-f` for everything.

Reply:
Ok, these should be fixed.